composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "valeryan/valet-wsl",
```

Comment:
fix any naming issues from your fork, please.

Reply:
Sure. Just changed it so I could get something up and running on a fork. Honestly this was a bit of a rush job to facilitate an immediate requirement and I wanted the upstream root CA stuff implemented in my installation quickly

Reply:
It happens often. When you plan to do a PR its best to create a separate branch for the PR so it doesn't interfere with what you are doing the fork for in the first place. You might want to move the changes you wish to merge into a branch like patch-1-ca-backport or something similar.